README.md

```diff
@@ -12,394 +12,61 @@
-and only exists to specify the correlation between on-chain data and off-chain items. There are three different variations of the `NFTKind` mode.
-
-1. `Physical`: The NFT represents a real-world physical item e.g., a house.
-2. `Digital`: The NFT represents a digital item, e.g., a unique JPEG or digital art.
-3. `Virtual`: The NFT is the virtual representation of a physical notion, e.g., a patent or copyright.
-
-The `NFTKind` mode is a required installation parameter and cannot be changed once the contract has been installed.
-The mode is passed in as a `u8` value to `nft_kind` runtime argument.
-
-| NFTKind  | u8  |
-| -------- | --- |
-| Physical | 0   |
-| Digital  | 1   |
-| Virtual  | 2   |
-
-#### NFTHolderMode
-
-The `NFTHolderMode` dictates which entities on a Casper network can own and mint NFTs. There are three different options currently available:
-
-1. `Accounts`: In this mode, only `Accounts` can own and mint NFTs.
-2. `Contracts`: In this mode, only `Contracts` can own and mint NFTs.
-3. `Mixed`: In this mode both `Accounts` and `Contracts` can own and mint NFTs.
-
-| NFTHolderMode | u8  |
-| ------------- | --- |
-| Accounts      | 0   |
-| Contracts     | 1   |
-| Mixed         | 2   |
-
-This modality is an optional installation parameter and will default to the `Mixed` mode if not provided. However, this
-mode cannot be changed once the contract has been installed.
-The mode is passed in as a `u8` value to `nft_holder_mode` runtime argument.
-
-#### Minting
-
-The minting mode governs the behavior of contract when minting new tokens. The minting modality provides two options:
-
-1. `Installer`: This mode restricts the ability to mint new NFT tokens only to the installing account of the NFT contract.
-2. `Public`: This mode allows any account to mint NFT tokens.
-3. `ACL`: This mode allows whitelisted accounts or contracts to mint NFT tokens.
-
-This modality is an optional installation parameter and will default to the `Installer` mode if not provided. However, this
-mode cannot be changed once the contract has been installed. The mode is set by passing a `u8` value to the `minting_mode` runtime argument.
-
-| MintingMode | u8  |
-| ----------- | --- |
-| Installer   | 0   |
-| Public      | 1   |
-| ACL         | 2   |
-
-#### WhitelistMode
-
-The `WhitelistMode` dictates if the ACL whitelist restricting access to the mint entry point can be updated. There are currently two options:
-
-1. `Unlocked`: The ACL whitelist is unlocked and can be updated via the set variables endpoint.
-2. `Locked`: The ACL whitelist is locked and cannot be updated further.
-
-If the `WhitelistMode` is set to `Locked` an ACL whitelist of entity keys must be provided on installation. This whitelist dictates which entities can mint NFTs in the restricted `ACL` minting mode. These entities include `Accounts` and/or `Contracts`.
```

Comment:
The information about ACL didn't transfer over to the modalities.md file.

Reply:
Good catch! Not sure what was going on there, but it's updated now.